pkg/network/ovn_kubernetes.go

```diff
@@ -160,16 +174,42 @@ func boostrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
 		return nil, fmt.Errorf("unable to bootstrap OVN, no master nodes found")
 	}
 
-	ovnMasterNodes := []string{}
+	var ovnMasters ovnMasterSlice
+
 	for _, masterNode := range masterNodeList.Items {
-		ovnMasterNodes = append(ovnMasterNodes, masterNode.Name)
+		var ip string
+		for _, address := range masterNode.Status.Addresses {
+			if address.Type == corev1.NodeInternalIP {
+				ip = address.Address
+				break
+			}
+		}
+		if ip == "" {
+			return nil, fmt.Errorf("No InternalIP found on master node '%s'", masterNode.Name)
+		}
+		parsedIP := net.ParseIP(ip)
+		if parsedIP == nil {
+			return nil, fmt.Errorf("Failed to parse InternalIP '%s' found on master node '%s'", ip, masterNode.Name)
+		}
+		if parsedIP.To4() == nil {
+			// IPv6, wrap address in brackets
+			ip = fmt.Sprintf("[%s]", ip)
```

Comment:
It's a little weird to call them "IPs" when they're "IPs with brackets around them", although I guess the alternative would end up being a lot messier.

Reply:
bracketedIP?  :-)